controllers/operator/clusterpodplacementconfig_controller.go

```diff
@@ -159,6 +161,33 @@ func (r *ClusterPodPlacementConfigReconciler) ensureNamespaceLabels(ctx context.
 	return err
 }
 
+func (r *ClusterPodPlacementConfigReconciler) ensureCorrectHostmountSCC(ctx context.Context) error {
+	// OpenShift introduces a new SCC `hostmount-anyuid-v2` with elevated privileges
```

Comment:
```suggestion
	// OpenShift 4.19 introduced a new SCC `hostmount-anyuid-v2` with elevated privileges
```

I'd also move the whole comment above the function definition to have it in the godoc